File: first_python.py
def get_first_python(users):
    new_arr = []
    for x in range(len(users)):
        if users[x]['language'] == 'Python':
            new_arr.append(users[x])
    first_name = ''
    country = ''
    if len(new_arr) > 0:
        first_name = new_arr[0]['first_name']
        country = new_arr[0]['country']
        return '{}, {}'.format(first_name, country)
    else:
        return 'There will be no Python developers'

File: first_python.js
function getFirstPython(list) {
    let newArr = [];
   for (let i = 0; i < list.length; i++){
     if (list[i].language == 'Python'){
       newArr.push(list[i]);
     }
   }
    let firstName;
    let country;
    if (newArr.length > 0){
       firstName = newArr[0].firstName;
        country = newArr[0].country;
      return `${firstName}, ${country}`;
    }
    return 'There will be no Python developers';
  }